domain_scoring/domain_scoring.py

```diff
@@ -1,17 +1,116 @@
 from explanation.explanation import Explanation
 from typing import List
 from main import RESEARCH_MODE
```

Comment:
I don't really now if thats such a good thing now, its a cyclic dependency...

Reply:
Fixed in #7